aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -590,7 +602,7 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
         // will have the chance to run their "beforeTextChanged" and "onTextChanged" with the same string!
 
         BlockElementWatcher(this)
-                .add(HeadingHandler())
+                .add(HeadingHandler(alignmentApproach))
                 .add(ListHandler())
                 .add(ListItemHandler())
                 .add(QuoteHandler())
```

Comment:
Do we want the same treatment to other block element spans, like the List and Quote? Gutenberg supports alignment for the Quote block (available on the UI) and the List block (albeit not exposed to the UI).

Reply:
As we discussed, I have updated all of the Spans that use alignment and extend `IAztecBlockSpan` to use this same approach for handling alignment.

> Gutenberg supports alignment for...the List block (albeit not exposed to the UI)

Is that correct? I don't see an alignment property [for list blocks](https://github.com/WordPress/gutenberg/blob/ef2ecc7fe27983a642b1ad027f7a0a2e12dff3e2/packages/block-library/src/list/block.json).

Reply:
> >
> >  Gutenberg supports alignment for...the List block (albeit not exposed to the UI)
> >
> Is that correct? I don't see an alignment property for list blocks.

I think it's supported via directly altering the markup. Try this block for example:
```
<!-- wp:list {"className":"wp-block-list has-text-align-center"} -->
<ul class="wp-block-list has-text-align-center"><li>Item 1</li><li>Item 2</li><li>Item 3</li></ul>
<!-- /wp:list -->
```